src/commands/dsyms/utils.ts

```diff
@@ -24,16 +26,25 @@ export const isZipFile = async (filepath: string) => {
   }
 }
 
-export const getMatchingDSYMFiles = async (absoluteFolderPath: string): Promise<Dsym[]> => {
-  const dSYMFiles = await globAsync(buildPath(absoluteFolderPath, '**/*.dSYM'))
+export const getMatchingDSYMFiles = async (
+  absoluteFolderPath: string,
+  context: BaseContext | undefined
```

Comment:
It'd be best to always define `context` and use a mock in the tests instead, cf [upload.test.ts from sourcemaps](https://github.com/DataDog/datadog-ci/blob/2ba2e690a11f5974c4c22c2fa36b0b9c8446b6cb/src/commands/sourcemaps/__tests__/upload.test.ts#L89-L90).